Snabble/UI/Onboarding/Model/OnboardingViewModel.swift

```diff
@@ -67,6 +79,7 @@ public final class OnboardingViewModel: ObservableObject, Codable {
     func next(for element: OnboardingItem) -> OnboardingItem? {
         guard let item = items.next(after: element) else {
             isDone = true
+            Onboarding.wasPerformed()
```

Comment:
Ich würde empfehlen es in didSet von `isDone` zu ändern. Falls wir zukünftig den Code ändern müssen wir immer Zeile 81 und 82 gemeinsam bewegen. Durch die Verwendung von `didSet` stellen wir sicher das weiterhin nur ein Wert geändert werden muss und alles funktioniert.

Reply:
88c2db861fc19a86fbd1d361163a6b9e857efe57